main/forum/editthread.php

```diff
@@ -0,0 +1,220 @@
+<?php
+/* For licensing terms, see /license.txt */
+
+/**
+ * Edit a Forum Thread
+ * @Author José Loguercio <jose.loguercio@beeznest.com>
+ *
+ * @package chamilo.forum
+ */
+
+use ChamiloSession as Session;
+
+// Including the global initialization file.
+require_once '../inc/global.inc.php';
+
+// The section (tabs).
+$this_section = SECTION_COURSES;
+
+// Notification for unauthorized people.
+api_protect_course_script(true);
+
+$nameTools = get_lang('ToolForum');
+
+// Unset the formElements in session before the includes function works
+unset($_SESSION['formelements']);
+
+/* Including necessary files */
+
+require_once 'forumconfig.inc.php';
+require_once 'forumfunction.inc.php';
+
+// Are we in a lp ?
+$origin = '';
+if (isset($_GET['origin'])) {
+    $origin = Security::remove_XSS($_GET['origin']);
+}
+
+/* MAIN DISPLAY SECTION */
+$current_forum = get_forum_information($_GET['forum']);
+$current_forum_category = get_forumcategory_information($current_forum['forum_category']);
+
+/* Breadcrumbs */
+
+if (isset($_SESSION['gradebook'])){
+    $gradebook = Security::remove_XSS($_SESSION['gradebook']);
+}
+
+if (!empty($gradebook) && $gradebook == 'view') {
+    $interbreadcrumb[] = array (
+        'url' => '../gradebook/'.Security::remove_XSS($_SESSION['gradebook_dest']),
+        'name' => get_lang('ToolGradebook')
+    );
+}
+
+if (!empty($_GET['gidReq'])) {
+    $toolgroup = intval($_GET['gidReq']);
+    Session::write('toolgroup',$toolgroup);
+}
+
+$threadId = isset($_GET['thread']) ? $_GET['thread'] : false;
```

Comment:
If $threadId is integer, please use intval() to make sure that the value is not abused by a hacker.